handbook/engineering/security/process.md

```diff
@@ -0,0 +1,64 @@
+# Security - How we manage our work
+
+Our processes are in a period of change, so if in doubt, please reach out to us directly in Slack at [#security](https://sourcegraph.slack.com/archives/C1JH2BEHZ).
+
+## Contents
+  * [Principles](#principles)
+  * [Process overview](#process-overview)
+  * [How to work with us](#how-to-work-with-us)
+  * [Other processes](#other-processes)
+
+## Principles
+
+1. [Goals](https://about.sourcegraph.com/company/goals/guidelines) are something we strive for, whilst tracking and communicating progress.
+2. A work item is a piece of work (e.g., writing code, hiring a new teammate) that makes progress toward achieving a goal.
+3. Releases may be made up of N workitems, that may impact Y goals. Whilst this is true, we communicate both internally and externally progress towards those goals.
+4. Security by its various nature has public work items ([main repo](https://github.com/sourcegraph/sourcegraph) and private workitems ([security repo](https://github.com/sourcegraph/security-issues/)). Over time workitems should move from the private repository to the public repository once they can be made public. The ideal goal state is the lack of a private security repository.
```

Comment:
Nit - Should we update this to reference Jira instead?

Reply:
Updating in branch and re-raising PR.